zuul-core/src/test/java/com/netflix/netty/common/proxyprotocol/ElbProxyProtocolChannelHandlerTest.java

```diff
@@ -95,9 +145,9 @@ public void negotiateProxy_ppv1_ipv4() {
 
     @Test
     public void negotiateProxy_ppv1_ipv6() {
-        ElbProxyProtocolChannelHandler handler = new ElbProxyProtocolChannelHandler(/* withProxyProtocol= */ true);
         EmbeddedChannel channel = new EmbeddedChannel();
-        handler.addProxyProtocol(channel.pipeline());
+        channel.pipeline().addLast(ElbProxyProtocolChannelHandler.NAME,
```

Comment:
When wrapping, it's a good idea to follow the rectangle rule:   https://github.com/google/google-java-format/wiki/The-Rectangle-Rule